bmlt-meeting-list.php

```diff
@@ -994,12 +995,21 @@ function bmlt_meeting_list($atts = null, $content = null) {
 									} elseif ( utf8_encode($this->mpdf->y) === $this->options['margin_top'] ) {
 										$header .= "<h2 style='".$header_style."'>".$this->getday($meeting_value['weekday_tinyint'], false, $this->options['weekday_language'])." " . $cont . "</h2>";
 									}
-									$header .= "<p style='margin-top:1pt; padding-top:1pt; font-weight:bold;'>".$area_name."</p>";
 									
-								} elseif ( utf8_encode($this->mpdf->y) === $this->options['margin_top'] ) {
-								
+									if ( $this->options['header_remove'] == 0 ) {
```

Comment:
this could be simplified:

```php
$header .= $this->options['header_remove'] == 1 ? "<p style='margin-top:1pt; padding-top:1pt; font-weight:bold;'>".$area_name."</p>" : ""
```

Reply:
thanks i actually almost made it more confusing than that too ha